src/server/scripts/Northrend/Naxxramas/boss_anubrekhan.cpp

```diff
@@ -211,12 +211,12 @@ class boss_anubrekhan : public CreatureScript
                         events.ScheduleEvent(EVENT_SPAWN_GUARD, 3 * IN_MILLISECONDS);
 
                         events.ScheduleEvent(EVENT_LOCUST_ENDS, RAID_MODE(19, 23) * IN_MILLISECONDS);
-                        events.ScheduleEvent(EVENT_LOCUST, 90000);
+                        events.ScheduleEvent(EVENT_LOCUST, 90 * IN_MILLISECONDS);
```

Comment:
Gonna stick with `uint32`s for now until the core supports `std::chrono` in more places (diff gets called with it etc).